Formula/tmux.rb

```diff
@@ -40,6 +23,11 @@ class Tmux < Formula
   depends_on "pkg-config" => :build
   depends_on "libevent"
 
+  resource "completion" do
+    url "https://raw.githubusercontent.com/przepompownia/tmux-bash-completion/678a27616b70c649c6701cae9cd8c92b58cc051b/completions/tmux"
+    sha256 "a0905c595fec7f0258fba5466315d42d67eca3bd2d3b12f4af8936d7f168b6c6"
+  end
```

Comment:
Oh, seems fine then :+1:. Could you ask them for a release tag and use the tarball from that? Thanks!